### DISCUSSION TITLE
Onboard en Debian 12  (bookworm)

### DISCUSSION THREAD

**Discussion by Enrike110** [deepin-community/.github#91] [General]
Good morning,
I have acquired Raspberry PI5 and it can only work with Debian 12, but I have a problem with Onboard, it installs correctly, but when I try to press any key, it closes and it does not overload the applications. So it changes the window when I try to type and try to select any key.

**Comment by BLumia** [comment]
Did you post it to the wrong place? FYI here is the deepin community, which is no longer a debian-based (package repo-wised) distro. Are you trying to use a deepin (instead of debian) system image for your raspberry pi?
